scissors/src/main/java/com/lyft/android/scissors/GlideFillViewportTransformation.java

```diff
@@ -49,11 +55,26 @@ protected Bitmap transform(BitmapPool bitmapPool, Bitmap source, int outWidth, i
     }
 
     @Override
-    public String getId() {
-        return getClass().getName();
+    public boolean equals(Object obj) {
```

Comment:
We really need to implement `equals()` and `hashCode()` according to the [`Transformation` documentation](https://github.com/bumptech/glide/blob/master/library/src/main/java/com/bumptech/glide/load/Transformation.java#L59) (and the use case mentioned before).